### ISSUE TITLE
Perform new accessibility audit

### ISSUE THREAD

**Issue by jennmalcolm** [ONRR/doi-extractives-data#4010]
Since we have transitioned to Gatsby, we need to run a new accessibility audit and document the issues.

**Comment by brentryanjohnson** [comment]
## Lighthouse results

- [id] attributes on the page are not unique - The value of an id attribute must be unique to prevent other instances from being overlooked by assistive technologies.

```
Failing Elements
--
rect#backgroundRect
g#maxExtent
g#bars
g#groups
```

- Lists do not contain only `<li>` elements and script supporting elements (`<script>` and `<template>`). Screen readers have a specific way of announcing lists. Ensuring proper list structure aids screen reader output.

```
Failing Elements
--
ul.header-nav_top
```

- `<html>` element does not have a `[lang]` attribute - If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.
```
Failing Elements
--
html
```

- Form elements do not have associated labels - Labels ensure that form controls are announced properly by assistive technologies, like screen readers.

```
Failing Elements
--
select
select
<select><option value="calendar_year">Calendar year</option><option selected="" value="fiscal_year">Fiscal year</option></select>
```

**Comment by brentryanjohnson** [comment]
## Pa11y results

 • Error: The html element should have a lang or xml:lang attribute which describes the language of the document.
   ├── WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
   ├── html
   └── <html><head><meta charset="utf-8"><me...</html>

 • Error: This select element does not have a name available to an accessibility API. Valid names are: label element, title attribute, aria-label attribute, aria-labelledby attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
   ├── #data-summary > section:nth-child(3) > div:nth-child(4) > div:nth-child(2) > div > select
   └── <select><option value="calendar_year">C...</select>

 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #data-summary > section:nth-child(3) > div:nth-child(4) > div:nth-child(2) > div > select
   └── <select><option value="calendar_year">C...</select>

 • Error: Duplicate id attribute value "backgroundRect" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #backgroundRect
   └── <rect id="backgroundRect" y="0" height="200" width="349" x="0" style="opacity: 0;"></rect>

 • Error: Duplicate id attribute value "maxExtent" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #maxExtent
   └── <g id="maxExtent"><text width="349" x="349" y="15...</g>

 • Error: Duplicate id attribute value "bars" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #bars
   └── <g id="bars"><g height="175" width="24.67676...</g>

 • Error: Duplicate id attribute value "groups" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #groups
   └── <g id="groups"><line x1="4.935353535353537" x2...</g>

 • Error: This select element does not have a name available to an accessibility API. Valid names are: label element, title attribute, aria-label attribute, aria-labelledby attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
   ├── #key-stats-production > div:nth-child(2) > div:nth-child(2) > div > select
   └── <select><option value="calendar_year">C...</select>

 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #key-stats-production > div:nth-child(2) > div:nth-child(2) > div > select
   └── <select><option value="calendar_year">C...</select>

 • Error: Duplicate id attribute value "backgroundRect" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #backgroundRect
   └── <rect id="backgroundRect" y="0" height="200" width="300" x="0" style="opacity: 0;"></rect>

 • Error: Duplicate id attribute value "maxExtent" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #maxExtent
   └── <g id="maxExtent"><text width="300" x="300" y="15...</g>

 • Error: Duplicate id attribute value "bars" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #bars
   └── <g id="bars"><g height="175" width="21.21212...</g>

 • Error: Duplicate id attribute value "groups" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #groups
   └── <g id="groups"><line x1="4.242424242424243" x2...</g>

 • Error: Duplicate id attribute value "backgroundRect" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #backgroundRect
   └── <rect id="backgroundRect" y="0" height="200" width="300" x="0" style="opacity: 0;"></rect>

 • Error: Duplicate id attribute value "maxExtent" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #maxExtent
   └── <g id="maxExtent"><text width="300" x="300" y="15...</g>

 • Error: Duplicate id attribute value "bars" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #bars
   └── <g id="bars"><g height="175" width="21.21212...</g>

 • Error: Duplicate id attribute value "groups" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #groups
   └── <g id="groups"><line x1="4.242424242424243" x2...</g>

 • Error: Duplicate id attribute value "backgroundRect" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #backgroundRect
   └── <rect id="backgroundRect" y="0" height="200" width="300" x="0" style="opacity: 0;"></rect>

 • Error: Duplicate id attribute value "maxExtent" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #maxExtent
   └── <g id="maxExtent"><text width="300" x="300" y="15...</g>

 • Error: Duplicate id attribute value "bars" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #bars
   └── <g id="bars"><g height="175" width="21.21212...</g>

 • Error: Duplicate id attribute value "groups" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #groups
   └── <g id="groups"><line x1="4.242424242424243" x2...</g>

21 Errors
